src_py/ftfont.py

```diff
@@ -32,19 +31,19 @@ class Font(_Font):
     __get_default_resolution = staticmethod(get_default_resolution)
     __default_font = encode_file_path(get_default_font())
 
-    __unull = as_unicode(r"\x00")
-    __bnull = as_bytes("\x00")
+    __unull = "\x00"
+    __bnull = b"\x00"
```

Comment:
This probably works, but I just want to confirm. __unull was a raw string and now it isn't.

Reply:
It's a pattern all over the codebase, it makes sense that a pygame compatibility method like `as_unicode` would want a raw string, to control encoding.